controllers/helpers.go

```diff
@@ -54,7 +54,8 @@ const (
 
 	gpuUnused = "UNUSED"
 
-	pollingInterval = time.Second * 2
+	taskPollingInterval   = 5 * time.Second
+	taskCompletionTimeout = 5 * time.Minute
```

Comment:
Is this the maximum time a control loop could run for? This seems a long time, especially if we only have 10 workers, meaning we can only detach volumes from 10 machines at a time. I'm concerned about the effect of this when there are lots of machines, either as large cluster or for many clusters. This would potentially affect clusters coming up and being upgraded as well as being deleted right?

Reply:
Is there any way we can filter tasks using https://developers.nutanix.com/api-reference?namespace=prism&version=v4.0.b1#tag/Tasks/operation/listTasks so we don't have to wait here at all?

Reply:
let's avoid batching for now. I'd like to familiarize myself w/ those APIs a bit more before committing CAPX to rely on them.

Reply:
as discussed offline, let's omit waiting altogether and just requeue for the time being